src/core/style/mixins.scss

```diff
@@ -326,3 +326,47 @@
     }
   }
 }
+
+// Only use when in row layout
+@mixin when-layout-row($element) {
```

Comment:
This seems like a whole lot of selectors (close 2kb), for something as simple as drawing a divider.